### PR TITLE
Fix styling issues with the tutorial

### LINKED ISSUE #634: Design review of tutorial
**Package**

lib-classifier

**Feature or Issue Description**

@beckyrother provided the following design notes:

Screenshot_3_29_19__11_38_AM

**To Dos**

- [ ] Adjust size of step buttons
- [ ] Reduce padding

### PR DESCRIPTION
**Pull request by srallen**
Package: lib-classifier, lib-grommet-theme

Closes #634 

Describe your changes:
The Grommet Paragraph component defaults to size medium, so I updated medium to use 14px and made a new set of sizes for small for use in our theme. 

The top padding issue was actually an hr element that wasn't rendering full width. I'm not sure why it was doing that, but the tutorial has some styling to fix it. I'm not sure if it'll be an issue in other components using markdown.

I've set the headings sizes in the theme.

I changed the image to use object-fit cover so it'll use the full width of the slide area.

I've decreased the radio button icon size in the theme. 

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

File: packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tabs/Tabs.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ function Tabs (props) {
     },
     tabs: {
       extend: `
-        button {
+        button[role="tab"] {
```
Comment (srallen) [inline packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tabs/Tabs.js]: This was breaking the radio button styling for the tutorial, so it needed greater specificity.

**Comment by srallen** [comment]
The design shows a standard heading showing the project name in the tutorial which I haven't implemented yet: https://projects.invisionapp.com/d/#/console/12923997/335563177/preview

There was discussion on slack about how to handle styling of the heading when the project owners could submit markdown with h1s in it. I still need to do some custom styling to handle this.

**Comment by srallen** [comment]
The modal heading is fixed in #655

**Comment by srallen** [comment]
This is ready for final review

File: packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js [hunk 2]
```diff
@@ -1,79 +1,70 @@
...
-    activeStep,
-    stepWithMedium
-  }
-}
-
 const StyledMarkdownWrapper = styled(Box)`
-  height: 100%;
-  max-height: ${props => props.isThereMedia ? '150px' : '100%'} ;
-`
+  > hr {
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js]: We should also add an `hr` to the storybook example markdown

File: packages/lib-classifier/src/components/Classifier/components/ModalTutorial/ModalTutorial.spec.js [hunk 3]
```diff
@@ -31,15 +31,17 @@ describe('ModalTutorial', function () {
...
+  //     <ModalTutorial.wrappedComponent
+  //       loadingState={asyncStates.success}
+  //       setModalVisibility={() => { }}
+  //       tutorial={tutorial}
+  //     />
+  //   )
 
-    expect(wrapper.find(SlideTutorial)).to.have.lengthOf(1)
-  })
+  //   expect(wrapper.find(SlideTutorial)).to.have.lengthOf(1)
```
[the remark this answers is not in the document]
Comment (srallen) [reply packages/lib-classifier/src/components/Classifier/components/ModalTutorial/ModalTutorial.spec.js]: The issue here isn't the ResponsiveContext in SlideTutorialContainer, but the one in the ModalTutorial. I'm not sure if it's worth writing another HOC to get this single test working.

**Pull request merged**